tests/validation/tests/v3_api/common.py

```diff
@@ -1,3 +1,4 @@
+from ..common import *  # NOQA
```

Comment:
Will all files referencing the deleting methods in here work? because we added this line now? Or should this line be made available in the other files too?

Reply:
`tests/common.py` is imported in `tests/v3_api/common.py`, so any file referencing `v3_api/common.py` will get those methods from the parent `common.py`

Reply:
I guess you mean the [remove_resource method](https://github.com/rancher/rancher/blob/2dca0c67b48331c23c05c025208bacfc258d45bd/tests/validation/tests/v3_api/conftest.py#L20-L47) in the conftest.py file? 

It is part of the pytest framework and is not affected by this change for methods in v3_api.

If we want to do the same thing for v1_api, then we need to have another conftest.py file in v1_api. It is not in this PR, we can do it in the future.

Reply:
Brandon clarified the question I had